tests/test_nginx_config_reloader.py

```diff
@@ -15,10 +15,19 @@ def setUp(self):
         self.get_pid.return_value = 42
 
         self.source = mkdtemp()
-        nginx_config_reloader.DIR_TO_WATCH = self.source
         self.dest = mkdtemp()
+        self.backup = mkdtemp()
+        _, self.mag_conf = mkstemp(text=True)
+        _, self.mag1_conf = mkstemp(text=True)
+        _, self.mag2_conf = mkstemp(text=True)
```

Comment:
No, that doesn't work, because separate testcases write to those files, and close them. So they are deleted after one test. It would probably be better if each testcase created it's own tmpfiles, but I'm reluctant to rewrite the whole thing.